entry.c

```diff
@@ -218,10 +218,16 @@ load_entry(FILE *file, void (*error_func)(), struct passwd *pw, char **envp) {
 		/* DOM (days of month)
 		 */
 
-		if (ch == '*')
-			e->flags |= DOM_STAR;
-		ch = get_list(e->dom, FIRST_DOM, LAST_DOM,
-			      PPC_NULL, ch, file);
+		if (ch == '$') {
+			e->flags |= DOM_LAST;
+			Skip_Nonblanks(ch, file);
```

Comment:
I think this will miss syntax errors after the '$'.  For example:
```
* * $foo * * echo testing
```
It seems safest to require that the next character after '$' be a blank.

Reply:
> I think this will miss syntax errors after the '$'. For example:
> 
> ```
> * * $foo * * echo testing
> ```
> 
> It seems safest to require that the next character after '$' be a blank.

This seems good to me, see the  Skip_Nonblanks and Skip_Blanks, they
will eat every charater and every whitespace after '$'.

Reply:
ok.